src/Microsoft.DotNet.XHarness.TestRunners.Xunit/WasmApplicationEntryPoint.cs

```diff
@@ -27,6 +27,11 @@ public abstract class WasmApplicationEntryPoint : WasmApplicationEntryPointBase
 
     protected override TestRunner GetTestRunner(LogWriter logWriter)
     {
+#if USE_XUNIT_V3
+        var runner = new XUnitTestRunner(logWriter);
+        ConfigureRunnerFilters(runner, ApplicationOptions.Current);
+        return runner;
+#else
```

Comment:
We don't need to do wasm just yet, simply throw a not supported exception if using v3.